srcpkgs/steamguard-cli/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'steamguard-cli'
+pkgname=steamguard-cli
+version=0.7.0
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+short_desc="A linux utility for generating 2FA codes for Steam and managing Steam trade confirmations."
```

Comment:
This isn't going to pass the linter, because the description is too long. It has to have 72 characters at most. My suggestion is something similar to "Utility for Steam 2FA and for managing trade confirmations" at 59 characters.